ui/app/src/services/content.ts

```diff
@@ -131,6 +131,38 @@ export function fetchContentInstance(
   return fetchContentDOM(site, path).pipe(map((doc) => parseContentXML(doc, path, contentTypesLookup, {})));
 }
 
+export function writeContent(
+  site: string,
+  path: string,
+  user: string,
+  fileName: string,
+  content: string,
+  unlock: boolean = true
```

Comment:
Please check if `user` is optional for the service. Let's try to modify the abstraction to be like this:
```suggestion
  site: string,
  path: string,
  content: string,
  options?: { user?: string, unlock?: boolean }
```